Aptoide/aptoide-client-v8
# [MOB-263] MigratedApp to room

**What does this PR do?**

   This PR aims at migrating the MigratedApp table to room.

**Database changed?**

   No

**Where should the reviewer start?**

- [x] RoomMigratedApp.java

**How should this be manually tested?**

Migrate an app with promotion (example cats) and after migrating check the promotion (which should only appear if you made the migration). Also you can add a breakpoint on AppViewManager loadAppcPromotion method and check the isMigrated condition.

**What are the relevant tickets?**

  Tickets related to this pull-request: [MOB-263](https://aptoide.atlassian.net/browse/MOB-263)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new keys, etc.) 



**Code Review Checklist**
